disdrodb/readers/DELFT/parser_RASPBERRY.py

```diff
@@ -395,80 +396,92 @@ def df_sanitizer_fun(df, lazy=False):
                                       verbose=verbose,
                                       debugging_mode=debugging_mode)
 
-            ##------------------------------------------------------.
-            #### - Read all raw data files into a dataframe  
-            df = read_L0_raw_file_list(file_list=file_list,
-                                       column_names=columns_names_temporary,
-                                       reader_kwargs=reader_kwargs,
-                                       df_sanitizer_fun=df_sanitizer_fun,
-                                       lazy=lazy,
+            for filepath in file_list:
+
+                try:
+                    suf = Path(filepath).stem
```

Comment:
Maybe nice variable name rather than suffix? 
file_suffix?
Add a comment like  # Define file suffix (i.e <actual pattern you extract>)

Reply:
I added a `log.info(...)` to print the file's suffix at every iteration!